server/src/main/java/org/elasticsearch/search/fetch/FetchPhase.java

```diff
@@ -148,22 +172,24 @@ public void execute(SearchContext context) {
                 throw new FetchPhaseExecutionException(context.shardTarget(), "Error running fetch phase for doc [" + docId + "]", e);
             }
         }
+        for (FetchSubPhaseProcessor processor : processors) {
```

Comment:
Do we need this new method `done`? It looks like the only implementation adds the debug info, which seems like it could be done when initializing the processor:

```
    @Override
            public void done() {
                breakdown.debug = delegate.getDebugInfo();
            }
```

Reply:
I'll double check. I sure thought I needed it but I may not.

Reply:
You were right. I didn't need it.